api/enki.py

```diff
@@ -385,8 +477,9 @@ def patron_activity(self, patron, pin):
                 raise CirculationException()
         for loan in result['checkedOutItems']:
             yield self.parse_patron_loans(loan)
-        for hold in result['holds']:
-            yield self.parse_patron_holds(hold)
+        for type, holds in result['holds'].items():
+            for hold in holds:
+                yield self.parse_patron_holds(hold)
```

Comment:
The Enki client is also missing holds support, so this doesn't matter yet, but when there are patron holds the old code crashes.